src/DOM.js

```diff
@@ -10,6 +10,7 @@ const util = require('util');
 const {defaultCanvasSize} = require('./constants');
 const {Event, EventTarget, MouseEvent} = require('./Event');
 const GlobalContext = require('./GlobalContext');
+const bindings = require('../native-bindings');
```

Comment:
Per Slack, probably don't do this. Native bindings is somewhat sensitive to when you require it, because of `Worker` threading.

https://github.com/webmixedreality/exokit/blob/0475da06f10938bdc2525ea3d8056ab6df3e2f4a/core.js#L1695

But that might be fixed when we move to Node's new `worker_threads`, which we should when it's stable.

Reply:
Then again, https://nodejs.org/api/worker_threads.html:

> Native addons are not supported yet.

Probably for similar reasons.